README.rst

```diff
@@ -116,7 +116,7 @@ The proxy index works out of the box in Volto.
 However, in Volto a direct (and much faster) search is possible by using the ``@kitsearch`` endpoint, bypassing the catalog.
 The endpoint takes a native Open-/ ElasticSearch query and returns the results with Plone permission check.
 
-The Volto add-on `volto-searchkit-block <https://github.com/rohberg/volto-searchkit-block/>`_ (based on `react-searchkit <https://www.npmjs.com/package/react-searchkit>`_) provides a configurable block using this endpoint.
+The Volto add-on `@rohberg/volto-searchkit-block <https://github.com/rohberg/volto-searchkit-block/>`_ (based on `react-searchkit <https://www.npmjs.com/package/react-searchkit>`_) provides a configurable block using this endpoint.
```

Comment:
We prefer to skip naming companies outside of the Contributors paragraph. And even there I usually put names only. One needs to follow the link anyway to get more information and the reference to Rohberg gets resolved. 
I know, in the Volto community companies are keen to have their name somewhere in the namespace, but this here is a backend package.